### ISSUE TITLE
Example regression GNN architecture for homogeneous graph and node level prediction

### ISSUE THREAD

**Issue by MICMTS** [pyg-team/pytorch_geometric#9041]
### 🚀 The feature, motivation and pitch

Hello,

I would like to know if there is any example/tutorial available to build a GNN model with Pytorch Geometric for a regression task using homogeneous graph and node level prediction?

I did not understood fully the suggestion in: https://github.com/pyg-team/pytorch_geometric/issues/3794

Thank you in advance!

**Comment by rusty1s** [comment]
I can't think of an example on top of my head, but usually it is not hart to convert a node classification task to a node regression task. It usually only involves changing the loss function.
